### PR TITLE
feat(get-issuances): create get issuances in counterparty

### PR DESCRIPTION
**Pull request by leeran7**
Added `getIssuances` method into `CounterPartyClient`
Added `Issuance` type

### CODE REVIEW

File: src/types/Issuance.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  issuer: string;
+  transfer: number;
+  callable: number;
+  call_date: number;
+  call_price: number;
+  description: string;
+  fee_paid: number;
+  status: string;
+  locked: number;
+  asset_longname: string | null;
```
Comment (leeran7) [inline src/types/Issuance.ts:19]: `string | null` seems like the only type that's correct here. I only saw nulls in Insomnia testing.

File: src/CounterpartyClient.ts [hunk 2]
```diff
@@ -10,6 +10,7 @@ import {
   MempoolMessage,
   Message,
   Order,
+  Issuance,
 } from './types';
 import { toBase64 } from './lib';
 import { Asset } from './types/Asset';
```
Comment (AryanJ-NYC) [inline src/CounterpartyClient.ts]: Hey. This is a Boy Scout rule kind of thing and doesn't really concern your PR (but would totally make me happy).

Can you merge line 16 and 17 into the `import from './types` above? There's no reason it should be on its own line.

Feel free to decline as this isn't necessarily part of your PR. If you find the time, would be nice.

File: src/CounterpartyClient.ts [hunk 3]
```diff
@@ -185,7 +194,8 @@ type Method =
   | 'get_blocks'
   | 'get_running_info'
   | 'get_element_counts'
-  | 'get_unspent_txouts';
+  | 'get_unspent_txouts'
+  | 'get_issuances';
```
Comment (AryanJ-NYC) [inline src/CounterpartyClient.ts]: This is repeated. I already included this like 19 lines above this.

**Review by AryanJ-NYC — CHANGES_REQUESTED** [review]

**Comment by AryanJ-NYC** [comment]
Ah, @leeran7, one more thing! On a fresh commit (that is, nothing in staging or changed), please run the command `npm version minor && git push --follow-tags`. Thanks.

File: src/CounterpartyClient.ts (hunk 3 above)
Comment (leeran7) [reply src/CounterpartyClient.ts]: Ahhhh. Didn't notice that.

File: src/CounterpartyClient.ts (hunk 2 above)
Comment (leeran7) [reply src/CounterpartyClient.ts]: You got it.

**Pull request merged**